tests/php/tests/includes/rest-api/test_class.wp-job-manager-controllers-status.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+class WP_Test_WP_Job_Manager_Controllers_Status extends WPJM_REST_TestCase {
+
+	/**
+	 * @var int
+	 */
+	private $admin_id;
+	/**
+	 * @var int
+	 */
+	private $default_user_id;
+
+	function setUp() {
+		parent::setUp();
+		$admin = get_user_by( 'email', 'rest_api_admin_user@test.com' );
+		if ( false === $admin ){
+			$this->admin_id = wp_create_user(
+				'rest_api_admin_user',
+				'rest_api_admin_user',
+				'rest_api_admin_user@test.com' );
+			$admin = get_user_by( 'ID', $this->admin_id );
+			$admin->set_role( 'administrator' );
+		}
+
+		$this->default_user_id = get_current_user_id();
+		$this->login_as_admin();
+	}
+
+	function login_as_admin() {
```

Comment:
Is it worth adding any of these helper functions and/or `setUp` logic to a parent class?